dll/win32/browseui/shellbrowser.cpp

```diff
@@ -319,14 +318,14 @@ class CShellBrowser :
         return CComObjectRootEx<CComMultiThreadModelNoCS>::InternalRelease();
     }
 #endif
-
+    BrowseUISettings m_settings;
```

Comment:
Can't it be kept amongst the other private members of the CShellBrowser class above?

Reply:
Moved.